app/controllers/web/dashboard/users_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Web
+  module Dashboard
+    # Assigns the role to the user
+    class UsersController < BaseController
+      before_action :user_find, except: :index
+      before_action :authorize_role
+      rescue_from Pundit::NotAuthorizedError, with: :redirect_to_dashboard_users
+      rescue_from CustomErrors::LastRoleError, with: :redirect_to_dashboard_user
```

Comment:
`CustomErrors` - quite unclear name
Let's namespace this exception same as the corresponding operation `Ops::Developer::UnassignRole::LastRoleError`

Reply:
You can even handle this exception at `remove_role` method, without global filtering.

Reply:
fix